site/layouts/servermessage.hbs

```diff
@@ -12,7 +12,7 @@
 		<meta name="robots" content="noindex, nofollow, noarchive" />
 		{{>headresources}}
 	</head>
-	<body vocab="http://schema.org/" typeof="WebPage">
+	<body vocab="https://schema.org/" typeof="WebPage">
```

Comment:
The schema.org vocabulary should use "http://" scheme, not the "https://" 

reference: https://www.w3.org/2011/rdfa-context/rdfa-1.1

Reply:
See my other conversation comment.

Reply:
Reverted this change as per our follow-up discussion.